### PR TITLE
[support] Apps: added deprecation messaging + removed pages

### PR DESCRIPTION
**Pull request by tanushree-sharma**
Deprecation messaging for Cloudflare Apps + removed pages that are no longer relevant

### CODE REVIEW

**Review by jedecf — COMMENTED** [review]
LGTM overall. But why are we keeping:
[Managing Cloudflare Apps](https://8d49e7f5.cloudflare-docs-7ou.pages.dev/support/more-dashboard-apps/cloudflare-apps/managing-cloudflare-apps/)
[Removing Cloudflare Apps](https://8d49e7f5.cloudflare-docs-7ou.pages.dev/support/more-dashboard-apps/cloudflare-apps/removing-cloudflare-apps/)
[Reporting bugs or feature requests for Cloudflare Apps](https://8d49e7f5.cloudflare-docs-7ou.pages.dev/support/more-dashboard-apps/cloudflare-apps/reporting-bugs-or-feature-requests-for-cloudflare-apps/)
[Troubleshooting issues with Cloudflare Apps](https://8d49e7f5.cloudflare-docs-7ou.pages.dev/support/more-dashboard-apps/cloudflare-apps/troubleshooting-issues-with-cloudflare-apps/)

As they seem to not be relevant content now that Apps is not available to use within dash.cloudflare.com ?

**Comment by tanushree-sharma** [comment]
@jedecf Users still see their installed apps on dash.cloudflare.com. We don't allow any new installs, but they have the ability to edit existing apps or uninstall them.  

Keeping the minimum pages around to reflect that.

**Comment by ngayerie** [comment]
Hi @tanushreesharma-cf 
Would it be possible for you to add redirects for the pages that are getting deleted?

**Comment by tanushree-sharma** [comment]
@ngayerie I'm not familiar with how to set up the redirects. @jedecf are you able to help out?

**Review by kodster28 — APPROVED** [review]

File: content/support/More Dashboard Apps/Cloudflare Apps/_index.md [hunk 1]
```diff
@@ -5,7 +5,16 @@ weight: 3
 ---
  
 # Welcome to the Cloudflare Apps section
- 
+
+{{<Aside type="warning">}}
+**Cloudflare Apps are deprecated and will be sunset in the near future. As of January 2024, new app installs are no longer supported.**
```
Comment (kodster28) [inline content/support/More Dashboard Apps/Cloudflare Apps/_index.md]:
```suggestion

**Cloudflare Apps are deprecated and will be sunset in the near future. As of January 2024, new app installs are no longer supported.**
```

File: content/_redirects [hunk 2]
```diff
@@ -901,6 +901,11 @@
 /support/firewall/managed-rules-web-application-firewall-waf/migrating-from-waf-managed-rules-to-waf-managed-rulesets/ /waf/reference/migration-guides/waf-managed-rules-migration/ 301
 /support/firewall/tools/understanding-cloudflare-zone-lockdown/ /waf/tools/zone-lockdown/ 301
 /support/firewall/tools/understanding-cloudflare-user-agent-blocking/ /waf/tools/user-agent-blocking/ 301
+/support/more-dashboard-apps/cloudflare-apps/installing-cloudflare-apps/ //support/more-dashboard-apps/cloudflare-apps/ 301
```
Comment (kodster28) [inline content/_redirects]:
```suggestion
/support/more-dashboard-apps/cloudflare-apps/installing-cloudflare-apps/ /support/more-dashboard-apps/cloudflare-apps/ 301
```

**Pull request merged**